types/vega/vegapulseAir/meta.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Vegapulse Air 23",
+  "version": "1.0.0",
+  "manufacturer": "Vega",
+  "url": "https://www.vega.com/de-de/produkte/produktkatalog/fuellstand/radar/vegapuls-air-23",
```

Comment:
possibly link the English version

https://www.vega.com/en-us/products/product-catalog/level/radar/vegapuls-air-23